lua/telescope/builtin/files.lua

```diff
@@ -145,106 +145,126 @@ files.grep_string = function(opts)
   }):find()
 end
 
--- TODO: Maybe just change this to `find`.
--- TODO: Support `find` and maybe let people do other stuff with it as well.
-files.find_files = function(opts)
+local function which_find_command(opts)
+  local command = opts.find_command
+  if command then
+    return command[1]
+  end
+  if 1 == vim.fn.executable "fd" then
+    return "fd"
+  elseif 1 == vim.fn.executable "fdfind" then
+    return "fdfind"
+  elseif 1 == vim.fn.executable "rg" then
+    return "rg"
+  elseif 1 == vim.fn.executable "find" and vim.fn.has "win32" == 0 then
+    return "find"
+  elseif 1 == vim.fn.executable "where" then
+    return "where"
+  end
+end
+
+local function find_command_options(opts)
+  local command = which_find_command(opts)
   local find_command = opts.find_command
```

Comment:
You need to make adjustments here. If `find_command` is not passed in its nil and shortly after you add doing `table.insert` thats bad

Reply:
Or for me it doesnt even work. No config

Reply:
You're right